.github/AL-Go-Settings.json

```diff
@@ -22,6 +26,12 @@
     "NextMinor.yaml",
     "PublishToEnvironment.yaml"
   ],
+  "buildModes": [
+    "Default",
+    "Clean"
```

Comment:
I guess there is no need to build the "separated modules" in CLEAN mode is there?

Reply:
We have such gate internally, so why not have it here as well.

Reply:
But maybe the DevTools needs only the _Default_.

Reply:
Default + Clean